vendor/tektelic/t0006377-tempo.yaml

```diff
@@ -0,0 +1,77 @@
+name: TEKTELIC TEMPO Meeting Room Booking Solution
```

Comment:
The other devices (tempo, seal, breeze-v and stork) still have TEKTELIC in their name too.